reaper-adm-extension/src/reaper_adm/AppConfig.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#define JUCE_USE_CURL 0
+
+#ifndef JUCE_DISPLAY_SPLASH_SCREEN
+#define JUCE_DISPLAY_SPLASH_SCREEN 0
+#endif
+
+#ifndef JUCE_REPORT_APP_USAGE
+#define JUCE_REPORT_APP_USAGE 0
+#endif
+
+#define JUCE_USE_DARK_SPLASH_SCREEN 1
+
+#define JUCE_MODULE_AVAILABLE_juce_audio_basics          1
+#define JUCE_MODULE_AVAILABLE_juce_audio_devices         1
+#define JUCE_MODULE_AVAILABLE_juce_audio_formats         1
+#define JUCE_MODULE_AVAILABLE_juce_audio_processors      1
+#define JUCE_MODULE_AVAILABLE_juce_core                  1
+#define JUCE_MODULE_AVAILABLE_juce_cryptography          1
+#define JUCE_MODULE_AVAILABLE_juce_data_structures       1
+#define JUCE_MODULE_AVAILABLE_juce_events                1
+#define JUCE_MODULE_AVAILABLE_juce_graphics              1
+#define JUCE_MODULE_AVAILABLE_juce_gui_basics            1
+#define JUCE_MODULE_AVAILABLE_juce_gui_extra             1
+#define JUCE_MODULE_AVAILABLE_juce_opengl                1

```

Comment:
Do we need all of those modules?

Reply:
Probably not - just a lazy C+P - I'll try figure out what we're actually using.